exotic/exotic.py

```diff
@@ -1957,7 +1984,7 @@ def main():
                         pix_coords = wcs_hdr.world_to_pixel_values(tar_radec[0], tar_radec[1])
                         tx, ty = pix_coords[0].take(0), pix_coords[1].take(0)
 
-                    psf_data['target'][i] = fit_centroid(imageData, [tx, ty])
+                    psf_data['target'][i] = fit_centroid(imageData, [tx, ty], 0)
```

Comment:
Safe change to this call?

Reply:
Not sure what is meant - it's consistently changed at all points in the code.

Reply:
So the method signature was changed and every call to it was changed to correspond? Just checking.

Reply:
@mikeprimm @mikeprimm-mirific So the method signature was changed and every call to it was changed to correspond? Just another sanity check. (See earlier reply about library usage.)

Reply:
Yep - scanned through and changed all references, so should be consistent throughout, and I've run tests on all relevant paths.